### ISSUE TITLE
Support for Gatsby v3

### ISSUE THREAD

**Issue by Mrtenz** [universse/gatsby-plugin-workerize-loader#6]
It looks like Gatsby v3 is not supported by this plugin currently. Using it results in the following deprecation messages, and crashes the process.
```
(node:50629) [DEP_WEBPACK_COMPILATION_CACHE] DeprecationWarning: Compilation.cache was removed in favor of Compilation.getCache()
(node:50629) [DEP_WEBPACK_TEMPLATE_PATH_PLUGIN_REPLACE_PATH_VARIABLES_HASH] DeprecationWarning: [hash] is now [fullhash] (also consider using [chunkhash] or [contenthash], see documentation for details)
(node:50629) [DEP_WEBPACK_DEPRECATION_ARRAY_TO_SET_INDEXER] DeprecationWarning: chunk.files was changed from Array to Set (indexing Array is deprecated)
(node:50629) [DEP_WEBPACK_CHUNK_ENTRY_MODULE] DeprecationWarning: Chunk.entryModule: Use new ChunkGraph API

not finished Building development bundle - 5.768s

error Command failed with exit code 1.
```

**Comment by phuoccss** [comment]
Thanks for raising the issue. The upstream library, https://github.com/developit/workerize-loader, needs to be made compatible with Webpack 5, which Gatsby v3 depends on. There are unmerged PRs that address this. Unfortunately, there's nothing I can do at the moment.

**Comment by edsu** [comment]
Any update on this? It should probably be removed from Gatsby's documentation if it can't be upgraded.

**Comment by Mrtenz** [comment]
@edsu Looks like `workerize-loader` isn't being actively maintained unfortunately, so you're likely better off moving to a different solution if you need Gatsby v3.

**Comment by rafcontreras** [comment]
@Mrtenz , @universse , and @edsu, I've managed to modify the [workerize-loader](https://github.com/developit/workerize-loader) enough to have it work with [WebPack 5](https://github.com/webpack/webpack), thus allowing this plugin to work with Gatsby 3 with minimal modifications in the `gatsby-node` `onCreateWebpackConfig` step.

The maintainer has not responded to any pull requests so it looks like I'll create a package this weekend.

**Comment by Mrtenz** [comment]
Awesome. I'll try it out soon!

**Comment by phuoccss** [comment]
Hi. I published an alpha version. Please give it a try and report any issue that you have. Much thanks to @rafcontreras.

`yarn add gatsby-plugin-workerize-loader@alpha`

`npm install gatsby-plugin-workerize-loader@alpha`

**Comment by rafcontreras** [comment]
Hi @universse I've been using this in production for a while now. Any chance of a non-alpha release? At work they are bit uncomfortable of the tag.

**Comment by phuoccss** [comment]
Sure. Will make a release over the weekend

**Comment by phuoccss** [comment]
Released version 2.0.0.
